validator/Dockerfile

```diff
@@ -51,22 +51,16 @@ FROM ${CUDA_SAMPLE_IMAGE} AS sample-builder
 RUN mkdir /artifacts
 RUN cp /cuda-samples/vectorAdd /artifacts/vectorAdd
 
-FROM nvcr.io/nvidia/cuda:12.8.1-base-ubi9
+# The C/C++ distroless image is used as a base since the CUDA vectorAdd \
+# sample application depends on C/C++ libraries.
+FROM nvcr.io/nvidia/distroless/cc:v3.1.7-dev
 
-# Remove CUDA libs(compat etc) in favor of libs installed by the NVIDIA driver
-RUN dnf remove -y cuda-*
+USER 0:0
```

Comment:
Does validator need to run as root?

Reply:
Hmm it doesn't appear I have anything written in my notes concerning this (I made these changes some time ago). My best guess is that this was needed for the `driver-validation` container where we create the `/dev/char` symlinks. But it is definitely worth validating and removing this if not required.

Reply:
Sure, once you find out why it's needed and we are certain that there is no way around this, please update the PR description with the details

Reply:
we run `chroot` to check for host driver installation, which needs to be run with root user.